systemtest/src/test/java/io/strimzi/systemtest/LogSettingST.java

```diff
@@ -479,6 +482,138 @@ void testJSONFormatLogging() {
         assertThat(StUtils.checkLogForJSONFormat(eoPods, "user-operator"), is(true));
     }
 
+    @Test
+    @Order(14)
+    void testDynamicallySetEOloggingLevels() {
+        String eoName = KafkaResources.entityOperatorDeploymentName(CLUSTER_NAME);
+        Map<String, String> eoPods = DeploymentUtils.depSnapshot(eoName);
+
+        // change inline logging
+        InlineLogging il = new InlineLogging();
+        il.setLoggers(Collections.singletonMap("rootLogger.level", INFO));
```

Comment:
How do ou know you are changing it? Aran't `INFO` levels usually the defaults?

Reply:
The test are using `DEBUG`.